AUTHORS

```diff
@@ -17,9 +17,13 @@ Contributors:
 * Nick Tiskov <daymansmail@gmail.com>
 
 Code from other projects:
-* files src/qtsingleapplication/* src/lineedit/*
+* files src/app/qtlocalpeer/*
   copyright: Nokia Corporation
-  license: LGPL
+  license: mixed
+
+* files src/gui/lineedit.*
```

Comment:
IIRC this file has received many changes, but I guess that doesn't mean we could stop using their license?

Reply:
If the contributors didn't care to restrict their contributions they fall under that license.

Reply:
Of course, I didn't fully understand everything, but reading [GPL FAQ](https://www.gnu.org/licenses/gpl-faq.en.html), I see that the license applies to the whole program.
We may obtain, use, modify, and distribute the program under its license. We may combine several (possibly modified) programs into one and distribute it under a license that meets the conditions of all the combined source programs.
If we borrow some individual source files from a program, we are actually combining a modified version of that program with our program. So the entire resulting product must be distributed under a single license (or several license each of) that applies to the entire product (to all of its files).